libraries/lib-shuttlegui/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+#[[
+An embedded language for describing dialog layouts, and also alternative
```

Comment:
I don't agree that ShuttleGui should be referred to as a language

Reply:
In case we decide to push for `Shuttle2` I don't think it will be a part of this specific library. I think we will have something like `lib-declarative-ui` or `lib-shuttle2` :-)

Reply:
Yes, if there is a replacement, it will be in another library, and this one, deprecated